test/shrinkwrap.ts

```diff
@@ -467,3 +467,39 @@ test('repeat install with no inner shrinkwrap should not rewrite packages in nod
   const m = project.requireModule('is-negative')
   t.ok(m)
 })
+
+test('installing from shrinkwrap when using npm enterprise', async (t: tape.Test) => {
+  const project = prepare(t)
+
+  const opts = testDefaults({registry: 'https://npm-registry.compass.com/'})
+
+  await installPkgs(['is-positive@3.1.0'], opts)
+
+  const shr = await project.loadShrinkwrap()
+
+  t.deepEqual(shr, {
+    dependencies: {
+      'is-positive': '3.1.0'
+    },
+    packages: {
+      '/is-positive/3.1.0': {
+        resolution: {
+          integrity: 'sha1-hX21hKG6XRyymAUn/DtsQ103sP0=',
+          tarball: 'https://npm-registry.compass.com/i/is-positive/_attachments/is-positive-3.1.0.tgz'
```

Comment:
I think maybe we could use a relative path here if the tarball is from the same domain as the registry. So in this case just `tarball: '/i/is-positive/_attachments/is-positive-3.1.0.tgz'`